Externals/hidapi/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(hidapi)
+
+add_library(hidapi hidapi/hidapi.h)
```

Comment:
Shared libraries need special treatment later on to be copied next to the executable or have its rpath adjusted properly. Unless you build it as shared and show me that binaries working, please make it static :)